docs/quotes/README.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: docs
+title: Quotes
+permalink: /quotes
+---
```

Comment:
@franciscodr is it okay to leave this file in here even if I take it out of the sidebar?

Reply:
This file is autogenerated from the docs and we should remove it. Any useful here should appear in the package info